src/main/webapp/i18n/en/student-question.json

```diff
@@ -3,7 +3,19 @@
         "studentQuestion": {
             "created": "New question successfully created",
             "updated": "Question successfully updated",
-            "deleted": "Question successfully deleted"
+            "deleted": "Question successfully deleted",
+            "text": "Question text",
+            "creationDate": "Asked on",
+            "votes": "Votes",
+            "answers": "Answers",
+            "approvedAnswers": "Approved answers",
+            "exerciseOrLecture": "Exercise / Lecture",
+            "overview": {
+                "title": "Questions",
+                "noQuestions": "There are no questions for this course",
+                "showApproved": "show questions with approved answers",
+                "hideApproved": "hide questions with approved answers"
```

Comment:
For both `showApproved` and `hideApproved`: Those should also start with an upper case to be consistent with the German version.